docs/studies-hcq.md

```diff
@@ -52,42 +37,32 @@ As a side note, [Siddiqi and Mehra](https://www.ncbi.nlm.nih.gov/pmc/articles/PM
 
 [TOP](#top)
 
-</TabItem>
-<TabItem value="cohorts">
-
 ### Study Populations
 
+**Study Period**: January 1, 2020 - Present
+
+**Index date**: [COVID-19 Positive Date](/docs/cohorts#covid-19-positive-date)
+
 * **Entry Cohort**:
-  * Age at `COVID-19-positive-date` ≥ 18 years AND
-  * `COVID-19-positive-date` after Jan 1, 2020 AND
-  * `COVID-19-related hospitalization`
+  * Age at [COVID-19 Positive Date](/docs/cohorts#covid-19-positive-date) ≥ 18 years AND
+  * [COVID-19 Positive Date](/docs/cohorts#covid-19-positive-date) after Jan 1, 2020 AND
+  * [COVID-19 Related Hospitalization](/docs/cohorts#covid-19-related-hospitalization)
 * **Target Cohort**:
   * Entry Cohort AND
-  * `Inpatient-HCQ-use` AND NOT `Prior-HCQ-use` [1]
+  * [Inpatient HCQ Use](/docs/cohorts#inpatient-hcq-use) AND NOT [Prior HCQ Use](/docs/cohorts#prior-hcq-use) [1]
 * **Control Cohort**:
   * Entry Cohort AND
-  * NOT `Inpatient-HCQ-use` AND NOT `Prior-HCQ-use` [1]
-
-#### Definitions
-
-* `COVID-19-positive` [2]:
-  * Any clinical diagnosis [3] code in set `C19HCC Confirmed COVID-19 infection` OR
-  * `Positive` or `detected` result from any test in set `C19HCC SARS-CoV-2 Qualitative Laboratory Test` [4] OR
-  * `Positive` or `detected` result from any test in set `C19HCC SARS-related Qualitative Laboratory Test` [4]
-* `COVID-19-positive-date`: earliest date of `COVID-19-positive` [5]
-* `Hospitalization`:
-  * encounter with class or type like `inpatient encounter` AND
-  * encounter status like `completed`
-* `COVID-19-related-hospitalization`: [6]
-  * `COVID-19-positive-date` during `hospitalization` OR
-  * (`hospitalization` up to 14 days after `COVID-19-positive-date`) AND (any clinical or billing diagnosis of `Respiratory Condition` associated with the hospitalization [7])
-* `Prior-HCQ-use`:
-  * `Hydroxychloroquine` on admission medication list or outpatient medication list
-* `Inpatient-HCQ-use`:
-  * `Hydroxychloroquine` medication administration during `COVID-19-related-hospitalization`
-    * Includes any dose, form, duration of HCQ
-    * Includes any level of disease severity while on HCQ
-    * Includes patients on other medications
+  * NOT [Inpatient HCQ Use](/docs/cohorts#inpatient-hcq-use) AND NOT [Prior HCQ Use](/docs/cohorts#prior-hcq-use)
+
+### Outcome Definitions
```

Comment:
Suggest we identify the outcome cohorts but define them in the cohort page.

Reply:
The combination of two cohorts is a cohort, to be sure. But I'm not sure it is worth defining another cohort explicitly for something this simple.